autoload/neomake/makers/ft/haskell.vim

```diff
@@ -73,9 +73,12 @@ function! neomake#makers#ft#haskell#hdevtools() abort
     if !exists('s:uses_cabal')
         let s:uses_cabal = 0
         if executable('stack')
-            let rootdir = systemlist('stack --verbosity silent path --project-root')[0]
-            if empty(glob(rootdir . '/*.cabal'))
-                let s:uses_cabal = 1
+            let output = neomake#utils#systemlist(['stack', '--verbosity silent', 'path', '--project-root'])
+            if !empty(output)
+                let rootdir = output[0]
+                if empty(glob(rootdir . '/*.cabal'))
+                    let s:uses_cabal = 1
+                endif
```

Comment:
Well, I'm having trouble getting hdevtools working through any versions of this maker. Its working fine from the command line, but not through the maker. I'll keep banging on this for a bit and report back when I have something.